main/src/main/java/com/sedmelluq/discord/lavaplayer/source/youtube/YoutubeAudioSourceManager.java

```diff
@@ -386,18 +402,27 @@ private boolean determineFailureReason(HttpInterface httpInterface, String video
       }
 
       Map<String, String> format = convertToMapLayout(URLEncodedUtils.parse(response.getEntity()));
-      return determineFailureReasonFromStatus(format.get("status"), format.get("reason"), mustExist);
+
+      if (format.containsKey("player_response")) { // new format
+        JsonBrowser playerResponse = JsonBrowser.parse(format.get("player_response"));
+        JsonBrowser playabilityStatus = playerResponse.get("playabilityStatus");
+        String status = playabilityStatus.get("status").text();
+        String reason = playabilityStatus.get("reason").text();
+        return determineFailureReasonFromStatus(status, reason, mustExist);
+      } else {
+        return determineFailureReasonFromStatus(format.get("status"), format.get("reason"), mustExist);
+      }
     }
   }
 
   private boolean determineFailureReasonFromStatus(String status, String reason, boolean mustExist) {
-    if ("fail".equals(status)) {
+    if ("fail".equals(status) || "ERROR".equals(status) || "UNPLAYABLE".equals(status)) {
       if (("This video does not exist.".equals(reason) || "This video is unavailable.".equals(reason)) && !mustExist) {
         return true;
       } else if (reason != null) {
         throw new FriendlyException(reason, COMMON, null);
       }
-    } else if ("ok".equals(status)) {
+    } else if (status.equalsIgnoreCase("ok")) {
```

Comment:
```suggestion
    } else if ("ok".equalsIgnoreCase(status)) {
```
To follow seds coding standards